### ISSUE TITLE
Same namespace for 2 different classes

### ISSUE THREAD

**Issue by viviengaetan** [SyslogicNL/DoctrineJsonFunctions#15]
Hey @Hikariii,
First I want to thank you for this library.

When I'm installing from composer I have the following Warning:
```
Warning: Ambiguous class resolution, "Syslogic\DoctrineJsonFunctions\Query\AST\Functions\Postgresql\JsonSharpGt" was found in both "/.../vendor/syslogic/doctrine-json-functions/src/Query/AST/Functions/Postgresql/JsonSharpGtGt.php" and "/.../vendor/syslogic/doctrine-json-functions/src/Query/AST/Functions/Postgresql/JsonSharpGt.php", the first will be used.
```

When I take a look at it, you have two classes with the same name:
1. [JsonSharpGt.php](https://github.com/SyslogicNL/DoctrineJsonFunctions/blob/master/src/Query/AST/Functions/Postgresql/JsonSharpGt.php) named `JsonSharpGt`
2. [JsonSharpGtGt.php](https://github.com/SyslogicNL/DoctrineJsonFunctions/blob/master/src/Query/AST/Functions/Postgresql/JsonSharpGtGt.php) named `JsonSharpGt`

Since they get the same FUNCTION_NAME I'm not sure about what to do (I never used Postgresql). If you now what has to be done I can provide a PR :wink:

**Comment by Hikariii** [comment]
can you install with the master branch? https://github.com/SyslogicNL/DoctrineJsonFunctions/pull/16

**Comment by viviengaetan** [comment]
That's good:
```
Updating dependencies (including require-dev)                  
  - Removing syslogic/doctrine-json-functions (1.0.1)
  - Installing syslogic/doctrine-json-functions (dev-master 55ad3e5)
    Cloning 55ad3e5c184a493de1edaa79ab000b5216d99be1 from cache
```

Wouldn't changing function name a BC Break?

**Comment by Hikariii** [comment]
It didn't work in the first place, because that function name was a duplicate

**Comment by viviengaetan** [comment]
Good, thank you :wink:

**Comment by Hikariii** [comment]
fixed and released in version 1.1.0
